pkgs/development/python-modules/customtkinter/default.nix

```
@@ -17,20 +18,24 @@ buildPythonPackage {
   pyproject = true;
   disabled = pythonOlder "3.7";
 
-  src = fetchPypi {
-    inherit pname version;
-    hash = "sha256-/Y2zuvqWHJgu5gMNuoC0wuJYWGMHVrUTmG2xkRPY0gc=";
+  src = fetchFromGitHub {
+    owner = "TomSchimansky";
+    repo = "CustomTkinter";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-1g2wdXbUv5xNnpflFLXvU39s16kmwvuegKWd91E3qm4=";
   };
 
   nativeBuildInputs = [
     setuptools
-    wheel
+    tkinter
+  ];
+
+  propagatedBuildInputs = [
+    darkdetect
+    packaging
+    typing-extensions
   ];
-  buildInputs = [ tkinter ];
-  propagatedBuildInputs = [ darkdetect ];
 
```

Also lets run the tests of the package

```suggestion
  nativeCheckInputs = [ pytestCheckHook ];
```

---

Hey, could I get a bit of helping hand with the tests, part? I'm not exactly sure how to proceed forward because of this error

```console
@nix { "action": "setPhase", "phase": "pytestCheckPhase" }
Running phase: pytestCheckPhase
Executing pytestCheckPhase
============================= test session starts ==============================
platform darwin -- Python 3.11.9, pytest-8.2.2, pluggy-1.5.0
rootdir: /private/tmp/nix-build-python3.11-customtkinter-5.2.2.drv-0/source
configfile: pyproject.toml
collecting ... /nix/store/9aq6ppc6rdxhimwpqbhz0xicpq0k54cg-pytest-check-hook/nix-support/setup-hook: line 53: 84246 Trace/BPT trap: 5       /nix/store/23igamxv27fkjrnmx1lpwhqbhnacv6j3-python3-3.11.9/bin/python3.11 -m pytest
/nix/store/28j5bkydq5fpiy2pllanssqdnk65an0z-stdenv-darwin/setup: line 1665: pop_var_context: head of shell_variables not a function context
```

---

I just ran the test on Linux, and they are all failing pretty much with `no display name and no $DISPLAY environment variable` so we can just remove the pytestCheckHook as test are not going to work in a sandbox.